### PR TITLE
Rebuild for Python 3.7, GCC 7, R 3.5.1, openBLAS 0.3.2, Update to 2.0.16

### PR DESCRIPTION
**Pull request by xhochy**
Fixes #43 
Fixes #44 
Fixes #45 
Fixes #46

### CODE REVIEW

**Comment by xhochy** [comment]
@conda-forge-admin please rerender

**Comment by xhochy** [comment]
Depends on https://github.com/conda-forge/regex-feedstock/pull/49

**Comment by xhochy** [comment]
Requires `wheel>=0.32`.

**Comment by xhochy** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -39,7 +39,7 @@ requirements:
     - pathlib  # [py27]
     - ujson >=1.35
     - dill >=0.2,<0.3
-    - regex >=2017.4.5,<2017.12.1
+    - regex =2018.01.10
```
Comment (honnibal) [inline recipe/meta.yaml]: Thanks for your help with this feedstock!

I'm not sure about this pin though. We need to keep the regex pin aligned with the pip version of spaCy. Is there a reason you want to pin to this version specifically? I know the latest versions mess things up (although I haven't gotten to the bottom of why...)

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,39 +12,40 @@ source:
...
+    - murmurhash >=0.28,<1.1.0
+    - cymem >=2.0.0,<2.1.0
+    - preshed >=2.0.1,<2.1.0
+    - thinc >=6.12.0,<6.13.0
     - plac >=0.9.6,<1.0.0
     - pathlib  # [py27]
     - ujson >=1.35
     - dill >=0.2,<0.3
-    - regex >=2017.4.5,<2017.12.1
+    - regex =2018.01.10
```
Comment (honnibal) [inline recipe/meta.yaml]:
```suggestion
    - regex >=2017.4.5,<2017.12.1
```

File: recipe/meta.yaml (hunk 1 above)
Comment (xhochy) [reply recipe/meta.yaml]: This pin is actually pulled from the git tag for the spacy release the meta.yml here was adopted. So this should be the correct version.

**Pull request merged**